src/tests/data/pubkey/sphincsplus_wots.vec

```diff
@@ -0,0 +1,15 @@
+# This test data was generated using customly written tests using the
+# reference implementation of SPHINCS+ Round 3.1 (https://github.com/sphincs/sphincsplus/)
+#
```

Comment:
Yeah, I think this would be more consistent and single WOTS+ signatures are fast. Otherwise, we could drop the WOTS+ tests, since they are only Sphincs+ internal logic and the Sphincs+ tests test WOTS+ implicitly million fold. However, IMO additional (fast) tests never hurt ;)